Bookshelf/shared/build.gradle.kts

```diff
@@ -1,11 +1,14 @@
 import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
 
 plugins {
-    kotlin("multiplatform")
+    alias(libsx.plugins.kotlinMultiplatform)
+    alias(libsx.plugins.androidLibrary)
+    alias(libsx.plugins.kotlinSerialization)
+    // For some reason libsx.plugins.realm does not resolve directly so go through the provider
```

Comment:
Is this because you were using snapshot and it was not published on Gradle portal?

Reply:
Yeah. It cannot verify the version when the plugin is [already on the classpath](https://github.com/realm/realm-kotlin-samples/pull/48/files#diff-b4972298410326012e18a386b27f789db8ac499f6209ec41b9ecd8a412e510a8R8). I could remote the `classpath` entry now that we have a released version, but a bit tedious if we want to do back and forth between snapshot/non-snapshot.

Reply:
I left it as is to ease working with snapshots, but added a comment.